tests/jvm/src/test/scala/scala/meta/tests/semanticdb/TargetedSuite.scala

```diff
@@ -1144,4 +1144,22 @@ class TargetedSuite extends DatabaseSuite(SemanticdbMode.Slim) {
       )
     }
   )
+
+  symbols(
+    "class Foo[@specialized(Int) T](v: T)".stripMargin,
+    """|_empty_.Foo# => class Foo
+       |_empty_.Foo#(v) => param v: T
+       |  [0..1): T => _empty_.Foo#[T]
+       |_empty_.Foo#[T] => specialized typeparam T
```

Comment:
^

Reply:
Pro tip, use `targeted` and put the caret on the `T`. That allows you to write a test only for the `specialized` flag. Example https://github.com/scalameta/scalameta/blob/5f198e00ff3a2c6c949d757f4f6512788b9d126f/tests/jvm/src/test/scala/scala/meta/tests/semanticdb/TargetedSuite.scala#L613-L627